site/docs/numbers/porting/lnpChecker.mdx

```diff
@@ -0,0 +1,192 @@
+---
+id: lnpChecker
+title: Check number(s) portability
+slug: /numbers/guides/porting/lnpChecker
+description: How to check if a number is portable
+keywords:
+  - bandwidth
+  - numbers
+  - porting
+  - portin
+  - port-in
+  - port
+  - lnp
+  - portability
+  - checker
+  - check
+image: '@site/static/img/bw-icon.svg'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const accountId = "{accountId}";
+export const orderId = "{orderId}";
+
+export const Highlight = ({children, color}) => (
+    <span
+      style={{
+        backgroundColor: color,
+        borderRadius: '2px',
+        color: '#fff',
+        padding: '0.2rem',
+      }}>
+      {children}
+    </span>
+  );
+
+For a smoother porting experience, you can check to see if your numbers are portable to the Bandwidth Network, before submitting a port order.
+:::info
+The `/bulkPortins` API eliminates the need for the `/lnpchecker` API by sorting the list of TNs automatically into a set of port-in requests by Port Type. For more details follow the [How to Bulk Port](/docs/numbers/guides/porting/createBulkPortins).
+:::
+
+For all TNs in a `/portins` request, all of the following must be true:
+
+1. They all have the same Port Type (see the `Port Types` section below)
+2. They all have the same losing carrier (toll free telephone numbers all have the same losing RespOrg)
+3. They are all associated with the same billing TN and Service Address
+
+### Port Types
+
+Port Type is a categorization of how the TNs submitted to the /lnpchecker API will be handled by Bandwidth.
+
+| Port Type       | Description                               |                                   |
+|:----------------------------------|:-------------------------------------------------------|
+| MANUAL | `MANUAL` (for all port types starting with `MANUAL`) indicates that if the TNs cannot be ported automatically, the Bandwidth LNP team will work with the appropriate porting vendor or losing carrier to ensure completion of the port-in. The /portins API can be used to submit manual port-ins, which will be identified as such, and a Zendesk ticket will be automatically created to notify the Bandwidth LNP team. |
```

Comment:
Change "Zendesk" to "support" - I prefer not to indicate a vendor implementation in the guide for something like this when we can.